tribe-ext-community-currency-symbol/index.php

```diff
@@ -0,0 +1,402 @@
+<?php
+/**
+ * Plugin Name:     The Events Calendar Extension: Community Events Cost Currency Symbol
+ * Description:     Adds a currency symbol selector option to the Community Events form (must first be setup at wp-admin > Events > Settings > Community tab > Form Defaults section). Additionally, the event creator can override the currency symbol's position instead of using your site's default from wp-admin > Events > Settings > General tab.
+ * Version:         1.0.0
+ * Extension Class: Tribe__Extension__CE_Event_Cost_Currency_Symbol
+ * Author:          Modern Tribe, Inc.
+ * Author URI:      http://m.tri.be/1971
+ * License:         GPLv2 or later
+ * License URI:     https://www.gnu.org/licenses/gpl-2.0.html
+ */
+
+// Do not load unless Tribe Common is fully loaded.
+if ( ! class_exists( 'Tribe__Extension' ) ) {
+	return;
+}
+
+/**
+ * Extension main class, class begins loading on init() function.
+ */
+class Tribe__Extension__CE_Event_Cost_Currency_Symbol extends Tribe__Extension {
+
+	/**
+	 * Setup the Extension's properties.
+	 *
+	 * This always executes even if the required plugins are not present.
+	 */
+	public function construct() {
+		// Only works with/after version 4.5 as previous versions included the field
+		$this->add_required_plugin( 'Tribe__Events__Community__Main', '4.5' );
+
+		$this->set_url( 'https://theeventscalendar.com/extensions/community-events-cost-currency-symbol/' );
+	}
+
+	/**
+	 * Extension initialization and hooks.
+	 */
+	public function init() {
+		add_action( 'admin_init', array( $this, 'add_settings' ) );
+		add_action( 'tribe_events_community_section_before_cost', array( $this, 'display' ) );
+	}
+
+	/**
+	 * Add options to Tribe settings page.
+	 *
+	 * @see Tribe__Extension__Settings_Helper
+	 */
+	public function add_settings() {
+		require_once dirname( __FILE__ ) . '/src/Tribe/Settings_Helper.php';
+
+		// Setup fields on the settings page
+		$setting_helper = new Tribe__Extension__Settings_Helper();
+
+		// List of allowed symbols
+		$fields['tribe_ext_ce_cost_currency_symbols'] = array(
+			'type'            => 'text',
+			'label'           => esc_html__( 'Event Cost: Allowed Currency Symbols', 'tribe-extension' ),
+			'tooltip'         => esc_html__( 'Pipe-separated list of allowed currency symbols. (Once this option is set and saved, you will be able to pick the Default Currency Symbol for new events.) The drop-down will display these in the order you enter them. Example 1: $|€|£|Fr|¥|CNY|C$|AU$. Example 2: USD|EUR|GBP|JPY|CNY|CAD|AUD', 'tribe-extension' ),
```

Comment:
Allow me to summarize all of my feedback regarding the settings area. I was only trying to make two general points. These points apply to any PR that adds a setting:

1. Concise wording provides the best UX. Short tooltips and labels that do not contain _unnecessary_ details are used throughout our settings tabs. 
2. Compare anew new settings to other similar ones in the plugins. Ask yourself questions like: Do my settings follow the same standard? Is the wording I used similar? Does mine contain any extra info the other one doesn't, and if so is that extra info truly a _necessity_ for the user? Etc.

The above two considerations are something that we all must ponder when adding a setting. So I just wanted to highlight them here as well in case you had not heard them before. If I were you I would review the verbiage and layout of my new settings with both of those in mind, and see if I feel there are any revisions that need to be made. 

Once you are happy with it you should be all set. If there is anything you do feel needs rewording, it doesn't hurt to request a second brief code review of the wording just for someone else to spot spelling or code errors. 

> I think the examples are fine, especially since the characters a little harder than regular letters to enter correctly (might have to google for them). But if you have an aversion to them, they could be removed from the option and left in the documentation article.

Makes sense. In that case do you feel that both examples are needed then?

> Are you suggesting one per line instead of pipe-separated?

Since you have already implemented pipe-separated I would stick with that. No need to change logic at this point.